src/realm/ucx/ucp_context.cc

```diff
@@ -452,17 +452,17 @@ namespace UCP {
     return false;
   }
 
-  bool UCPWorker::ep_get(int target, int remote_dev_index, ucp_ep_h *ep) const
```

Comment:
Why removing the const? I do not think we modify the UCPWorker in this function?